### DISCUSSION TITLE
deta vue nodejs example pls

### DISCUSSION THREAD

**Discussion by suntong** [deta/cloud-docs#554] [Help] [answered]
Can someone show me the simplest way to host vue based nodejs app in deta please?

I combined 

https://docs.deta.sh/docs/micros/getting_started/  
https://vuejs.org/guide/quick-start.html  

together, but only got a "null" in my deployed site.

Please help. thx

**Comment by abdelhai** [comment, marked as answer]
we're releasing a new platform with native support for vue. signup for access here https://alpha.deta.space

**Reply by suntong** [reply]
OK, Requested & waiting for approval. Thanks for providing such wonderful free services!
